RDF.js

```diff
@@ -967,6 +967,16 @@ function importItem(newItem, node) {
 	
 	// type
 	var type = getFirstResults(node, [n.dc+"type", n.dc1_0+"type", n.dcterms+"type"], true);
+	
+	/**CUSTOM ITEM TYPE  -- Currently only Dataset **/
+	if (type && type.toLowerCase() == "dataset") {
+		if (newItem.extra) {
+			newItem.extra += "\ntype: dataset";
+		}
+		else newItem.extra = "type: dataset";
+	}
+
+
```

Comment:
Common no. General -- other RDF imports, including Zotero RDF, DC RDF, and Bibliontology RDF. Arguably those should all recognize the (valid) DC type dataset.